tests/test_loaders_dumpers/test_issue_429.py

```diff
@@ -0,0 +1,54 @@
+import os
+import unittest
+from rdflib import Graph, Literal
+from rdflib import Namespace
+from rdflib.namespace import RDF
+from linkml_runtime.dumpers import rdflib_dumper
+from linkml_runtime.loaders import yaml_loader
+from linkml_runtime.utils.schemaview import SchemaView
+from tests.test_loaders_dumpers import INPUT_DIR, OUTPUT_DIR
+from tests.test_loaders_dumpers.models.personinfo_test_issue_429 import Container
+
+
+class RdfLibPrefixTestCase(unittest.TestCase):
+    SCHEMA = os.path.join(INPUT_DIR, 'personinfo_test_issue_429.yaml')
+    DATA = os.path.join(INPUT_DIR, 'example_personinfo_test_issue_429_data.yaml')
+    OUT = os.path.join(OUTPUT_DIR, 'example_personinfo_test_issue_429_data.ttl')
+
+    ORCID = Namespace('https://orcid.org/')
+    personinfo = Namespace('https://w3id.org/linkml/examples/personinfo/')
+    SDO = Namespace('http://schema.org/')
+
+    def __init__(self, *args, **kwargs):
```

Comment:
you might want to consider setUp() here, see https://docs.python.org/3/library/unittest.html#organizing-test-code